### PR TITLE
[ADD][12.0] website_sale_product_sort: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to set a default sorting criteria for the e-commerce product
list.

It provides the standard criterias, but allows to extend the options via third
modules.

cc @Tecnativa TT20974

### CODE REVIEW

File: website_sale_product_sort/models/website.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            ('list_price asc', _('Catalog price: Low to High')),
+            ('name asc', _('Name - A to Z')),
+            ('name desc', _('Name - Z to A')),
+        ]
+
+    default_product_sort_criteria = fields.Selection(
+        selection='_get_product_sort_criterias',
+        string="Sort products by",
+        help="Default criteria for sorting products in the shop",
+        default='website_sequence asc',
```
Comment (sergio-teruel) [inline website_sale_product_sort/models/website.py]: This default value not exists, so this module can not be installed

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: website_sale_product_sort/models/website.py (hunk 1 above)
Comment (chienandalu) [reply website_sale_product_sort/models/website.py]: Sure. I changed the options in the last force push and forgot about the default. Thanks

**Review by sergio-teruel — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
:+1:

**Comment by ValentinVinagre** [comment]
The `'development_status': 'Beta'` it's correct?

**Comment by pedrobaeza** [comment]
Well, it can fit stable level:

https://odoo-community.org/page/module-maturity-levels

but we can let it as is if we want.

**Comment by ValentinVinagre** [comment]
> Well, it can fit stable level:
> 
> https://odoo-community.org/page/module-maturity-levels
> 
> but we can let it as is if we want.

okay.

**Pull request merged**